blivet/udev.py

```diff
@@ -164,11 +164,13 @@ def resolve_glob(glob):
         name = device_get_name(dev)
         path = device_get_devname(dev)
 
-        if fnmatch.fnmatch(name, glob) or fnmatch.fnmatch(path, glob):
+        if name and fnmatch.fnmatch(name, glob):
+            ret.append(name)
+        elif path and fnmatch.fnmatch(path, glob):
             ret.append(name)
         else:
             for link in device_get_symlinks(dev):
-                if fnmatch.fnmatch(link, glob):
+                if link and fnmatch.fnmatch(link, glob):
```

Comment:
Did you see evidence that the return value of device_get_symlinks can contain empty strings and/or None values?

Reply:
No, I didn't. The bug is reported only for `device_get_devname`, but it looked weird to fix only that one case.

Reply:
Fair enough. It doesn't seem as warranted as the others do, but it certainly shouldn't hurt anything either.